diffkemp/simpll/Transforms.cpp

```diff
@@ -114,6 +115,8 @@ std::vector<FunPair> simplifyModulesDiff(Config &config) {
     PassManager<Module, AnalysisManager<Module, Function *>, Function *,
         Module *> mpm;
     mpm.addPass(RemoveUnusedReturnValuesPass {});
+    if (config.ControlFlowOnly)
+        mpm.addPass(ReduceFunctionMetadataPass {});
```

Comment:
I wouldn't do this for control-flow-only, but always. Section attributes should not be relevant for the function semantics.